src/main/java/seedu/duke/Messages.java

```diff
@@ -43,10 +43,10 @@ public class Messages {
 
     public static final String MESSAGE_MISSING_SUB_COMMAND_TYPE = "OOPS!!! Please specify sub-command type.\n"
             + "For client: <command> -client\n"
-            + "For property: <command> -property";
+            + "For property: <command> -property\n";
```

Comment:
Good use of the next line to segregate different message to make it easier for the user to read.